wandb/proto/wandb_settings.proto

```diff
@@ -106,6 +106,7 @@ message Settings {
   ListStringValue ignore_globs = 78;
   google.protobuf.DoubleValue init_timeout = 79;
   google.protobuf.BoolValue is_local = 80;
+  google.protobuf.StringValue job_name = 145;
```

Comment:
Not sure if this is the right way to add a value or if I should set it to 81 and increment the rest of the numbers.

Reply:
For settings I'm not sure, we should ask @dmitryduev who appears to have just added this?

Reply:
the way you did it is great! we're not relying on field numbers anywhere internally